mmrazor/models/algorithms/base.py

```diff
@@ -39,6 +39,8 @@ class BaseAlgorithm(BaseModel):
             config of :class:`BaseDataPreprocessor`. Defaults to None.
         init_cfg (dict): The weight initialized config for
             :class:`BaseModule`.
+        module_inplace(bool): Whether to allow student module inplace
```

Comment:
This is the `base class` of all algorithms. Some algorithms (like NAS and Pruning) do not contain `student module`, so the docstring here need to be modified.

Reply:
I'm sorry for ignoring this problem. Now I have modified the document. Thank you very much!